README.md

```diff
@@ -1,71 +1,93 @@
-
-# Github repository:
-
-All design ideas, development decisions and product vision (milestones, releases), will happen in github.
-
-# Near Vision of Product (iteration one):
-
-# Far Vision of Product:
-
-Green Light Health App helps businesses prevent spread of illness in their locations
-
-# Stakeholders:
-- Venue owner
-- Concert goer
-- Gym user
-- Restaurant diner
-
-# Detailed Personas:
-Chris J
-Boston, MA
-Age 25
-An avid music fan since a young age, Chris has been starved for live music since the onset of the pandemic in mid-March.
-As a regular concert goer who enjoys all types of music, his favorite band the Red Hot Chili Peppers were due to perform
-at the 2020 Boston Calling music festival. Sadly, the event was postponed until further notice due to the global pandemic. As Red
-Hot Chili Peppers begin to plan their return to the Boston area next Summer, they will require attendees to download and install
-the Green Light Health app in order to confirm the health status of all concert guests. As the event approaches, Chris will
-need to use the Green Light Health web app and complete his health declaration so that he can attend the event and see his beloved
-Red Hot Chili Peppers.
- 
-James H
-Boston, MA
-Age 46
-James owns and operates "Ride The Lightning", a live music venue in Kenmore Square directly adjacent to Fenway Park. Due to
-the pandemic, James was forced to close his business until further notice, causing his entire revenue stream to dry up. However,
-as time went on and confirmed coronavirus cases began to drop off in Massachusetts, he began to envision how to reopen his club safely 
-and begin welcoming back his customers. He recently learned of the Green Light Health app and has determined that it will be required for
-all guests to download the app and show the green check to a club security guard in order to access the venue. As a promoter, 
-James needs the ability to create a business account for his venue and determine the specific health requirements that he will enforce
-in order to allow his customers access to the club. James needs this app to be accurate as possible, he cannot risk having an outbreak
-occur at his club because he does not believe the club will financially survive another shut down.
-    
-
-# Product Backlog:
-Find the [Product Backlog](https://github.com/jorgecotillo/green_light_health/projects/1)
-under github projects.
-
-# Definition of Ready:
-Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
-The below checklist should be complete to ensure a story is Ready.
-- [ ] The story captures and describes a reasonable unit of work
-- [ ] The amount of work to complete the story has been estimated using the approved measurement (T-shirt Size)
-- [ ] The story is in line with the Product Vision
-- [ ] All blockers and dependencies for the needed implementation are resolved
```

Comment:
@samNson With a whole file diff like this, it is hard to ascertain which lines have changed. Did you copy paste a new version of the file into your development branch?

Reply:
@nasadorian no i merged master into my  branch before i added my personas. Maybe that caused the issue.